test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -367,9 +366,11 @@ func TestAPIExportPermissionClaims(t *testing.T) {
 		if len(ul.Items) == 2 {
 			return true, "found two items"
 		}
-		return false, fmt.Sprintf("waiting on dynamic client to find both objects, found %v objects", len(ul.Items))
+		return false, fmt.Sprintf("waiting on dynamic client to find both objects, found %#v objects", ul.Items)
 	}, wait.ForeverTestTimeout, 100*time.Millisecond, "unable to wait for the two objects to be returned")
 
+	t.Logf("Items: %#v", ul.Items)
```

Comment:
no need to log this. It's logged by framework.Eventually

Reply:
updated